components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/Connection.kt

```diff
@@ -62,9 +66,14 @@ internal object RustPlacesConnection : Connection {
         return api!!.getWriter()
     }
 
-    override fun sync(syncInfo: SyncAuthInfo) {
+    override fun syncHistory(syncInfo: SyncAuthInfo) {
+        check(api != null) { "must call init first" }
+        api!!.syncHistory(syncInfo)
+    }
+
+    override fun syncBookmarks(syncInfo: SyncAuthInfo) {
         check(api != null) { "must call init first" }
```

Comment:
Unrelated to absolutely anything, is `check` standard? Dang, we should be using it in a-s...